### ISSUE TITLE
Many more modules now default-enabled

### ISSUE THREAD

**Issue by thijskh** [simplesamlphp/simplesamlphp#304]
In 1.14.0 RC1, many more of the included modules are now default enabled:

```
aggregator2/default-enable  consentSimpleAdmin/default-enable  metaedit/default-enable        papi/default-enable         sqlauth/default-enable
aggregator/default-enable   core/default-enable                modinfo/default-enable         portal/default-enable
authorize/default-enable    InfoCard/default-enable            multiauth/default-enable       saml2debug/default-enable
autotest/default-enable     ldap/default-enable                openid/default-enable          saml/default-enable
consent/default-enable      logpeek/default-enable             openidProvider/default-enable  sanitycheck/default-enable
```

I'm not sure this is intentional, but probably a side effect from moving them out to their own repository at a later point.

**Comment by jaimeperez** [comment]
My mistake indeed! Thanks for the heads-up Thijs!

**Comment by jaimeperez** [comment]
Fixed in 08e0f9b.
